views/check-your-answers.html

```diff
@@ -9,606 +9,202 @@
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Home address
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    72 Guild Street
-                    <br>London
-                    <br>SE23 6FH
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> home address</span>
-                    </a>
-                </dd>
-            </div>
-
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Occupation
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>Director</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> occupation</span>
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Role and responsibilities
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    <p>Strategic management of the company and the issuing of necessary directives</p>
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a href="/register/managing-officer">
-                        Change
-                        <span class="govuk-visually-hidden"> role and responsibilities</span>
-                    </a>
-                </dd>
-            </div>
-        </dl>
+      {% set formattedPrincipalAddressHtml %}
+        {% if appData.entity.principalAddress.principalAddressLine1 %}
+          {{ appData.entity.principalAddress.principalAddressLine1 }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressLine2 %}
+          <br>{{ appData.entity.principalAddress.principalAddressLine2 }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressTown %}
+          <br>{{ appData.entity.principalAddress.principalAddressTown }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressCounty %}
+          <br>{{ appData.entity.principalAddress.principalAddressCounty }}
+        {% endif %}
+        {% if appData.entity.principalAddress.principalAddressPostcode %}
+          <br>{{ appData.entity.principalAddress.principalAddressPostcode }}
+        {% endif %}
+      {% endset %}
+
+      {% set formattedServiceAddressHtml %}
```

Comment:
We need to create a generic template address as soon as I finish my work on roe-153. All address object will have the same data type